src/pages/timecard/Timecard.jsx

```diff
@@ -76,6 +76,9 @@ const Timecard = () => {
       {isAlertVisible && Object.keys(summaryMessages).length !== 0 && (
         <MessageSummary
           keys={sortErrorKeys(summaryMessages, desiredMessageOrder)}
+          messageSummary={summaryMessages}
+          setSummaryMessages={setSummaryMessages}
+          setIsAlertVisible={setIsAlertVisible}
```

Comment:
Do we need these? They aren't defined in the props of MessageSummary.

Reply:
These have been removed

Reply:
Looks like they're still here

Reply:
Thank you sir